docs/references/v2.openapi.json

```diff
@@ -956,11 +955,11 @@
           "401": {
             "$ref": "#/components/responses/401"
           },
-          "404": {
-            "$ref": "#/components/responses/404-schema-validation"
-          },
           "403": {
             "$ref": "#/components/responses/403-schema-validation"
+          },
+          "404": {
```

Comment:
Unrelated to this PR but why is it pointing to schema-validation?

Reply:
There are two 404 responses, one generic, one specific to schema-validation

Reply:
<img width="416" alt="Screenshot 2023-08-18 at 4 49 43 PM" src="https://github.com/liveblocks/liveblocks/assets/6947265/56ccc097-1e61-4571-9477-5e06266e051f">
<img width="319" alt="Screenshot 2023-08-18 at 4 49 33 PM" src="https://github.com/liveblocks/liveblocks/assets/6947265/63f22026-1fbd-4c47-90f7-6b2fd9b29576">